webapp/channels/src/components/custom_status/custom_status_emoji.tsx

```diff
@@ -48,6 +45,37 @@ function CustomStatusEmoji({
 
     const customStatusExpired = useSelector((state: GlobalState) => isCustomStatusExpired(state, customStatus));
     const customStatusEnabled = useSelector(isCustomStatusEnabled);
+
+    const [placement, setPlacement] = useState('bottom');
+    const emojiRef = useRef<HTMLSpanElement>(null);
+
+    useEffect(() => {
+        function handleMouseEnter() {
```

Comment:
Is there a reason this is nested within the `useEffect` and not a private member function?

Reply:
Yes to save on a dependency see "Needlessly externally defined functions" in https://www.epicreact.dev/myths-about-useeffect